core/controllers/suggestion.py

```diff
@@ -485,11 +485,24 @@ def _construct_exploration_suggestions(suggestions):
 
```

Comment:
Maybe in the docstring at the top, you could explain in the Returns value what sorts of things are omitted -- this is part of the contract of the function.

Shouldn't there be more tests added for this function (or, rather, functions which call this function)?

Reply:
Done